src/genie/libs/parser/iosxe/show_wireless.py

```diff
@@ -2053,3 +2053,935 @@ def cli(self, output=None):
+
+        # Webauth HTTP status counts
+        p_webauth_http_status_counts = re.compile(r"^Webauth\s+HTTP\s+status\s+counts$")
+
+        # Webauth backpressure queue counters
+        p_webauth_backpressure = re.compile(r"Webauth\s+backpressure\s+queue\s+counters")
+
+        # Dot1x Global Statistics
+        p_dot1x_global_section = re.compile(r"Dot1x\s+Global\s+Statistics")
+
+        # RxStart = 6657 RxLogoff = 18  RxResp = 104310 RxRespID = 15453
+        p_dot1x_rxstart = re.compile(r"^RxStart\s+=\s+(?P<start>\d+)\s+RxLogoff\s+=\s+(?P<logoff>\d+)\s+RxResp\s+=\s+(?P<resp>\d+)\s+RxRespID\s+=\s+(?P<respid>\d+)$")
+
+        # RxReq = 0 RxInvalid = 0  RxLenErr = 0
+        p_dot1x_req = re.compile(r"^RxReq\s+=\s+(?P<req>\d+)\s+RxInvalid\s+=\s+(?P<invalid>\d+)\s+RxLenErr\s+=\s+(?P<lenerror>\d+)$")
+
+        # RxTotal = 126939
+        p_dot1x_rx_total = re.compile(r"^RxTotal\s+=\s+(?P<total>\d+)$")
+
+        # TxStart = 0 TxLogoff = 0  TxResp = 0
+        p_dot1x_txstart = re.compile(r"^TxStart\s+=\s+(?P<start>\d+)\s+TxLogoff\s+=\s+(?P<logoff>\d+)\s+TxResp\s+=\s+(?P<resp>\d+)$")
+
+        # TxReq = 117345 ReTxReq = 3909  ReTxReqFail = 11528
+        p_dot1_tx_req = re.compile(r"^TxReq\s+=\s+(?P<req>\d+)\s+ReTxReq\s+=\s+(?P<rereq>\d+)\s+ReTxReqFail\s+=\s+(?P<txfail>\d+)$")
+
+        # TxReqID = 67670 ReTxReqID = 29908  ReTxReqIDFail = 1310
+        p_dot1_tx_reqid = re.compile(r"TxReqID\s+=\s+(?P<reqid>\d+)\s+ReTxReqID\s+=\s+(?P<rereqid>\d+)\s+ReTxReqIDFail\s+=\s+(?P<reqidfail>\d+)$")
+
+        # TxTotal = 185057
+        p_dot1_tx_total = re.compile(r"TxTotal\s+=\s+(?P<tx_total>\d+)$")
+
+        # Total client delete reasons
+        p_total_client_delete_reasons = re.compile(r"^Total\s+client\s+delete\s+reasons$")
+
+        # Controller deletes
+        p_controller_deletes = re.compile(r"^Controller\s+deletes$")
+
+        # Informational Delete Reason
+        p_informational_delete = re.compile(r"^Informational\s+Delete\s+Reason$")
+
+        # Client initiate delete
+        p_client_initiate_delete = re.compile(r"^Client\s+initiate\s+delete$")
+
+        # AP Deletes
+        p_ap_deletes = re.compile(r"^AP\s+Deletes$")
+
+        # [key] : [value]
+        p_colon_split = re.compile(r"^(?P<key>[\S\s]+\S)\s*: +(?P<value>\d+)$")
+
+        section_tracker = []
+        client_detail_dict = {}
+
+        for line in output.splitlines():
+          line = line.strip()
+          if p_total_client.match(line):
+              # Total Number of Clients : 16
+              match = p_total_client.match(line)
+              client_detail_dict.update({ "total_clients": int(match.group("value")) })
+              continue
+          elif p_protocol_stats_section.match(line):
+              # Protocol Statistics
+              client_detail_dict.update({ "protocol_statistics": {} })
+              section_tracker.append("protocol_statistics")
+              continue
```

Comment:
You are using ```continue``` for all the if condition, in that case having conditions as ```elif``` seems unnecessary. I suggest you change those elif's to simply if and keep the continues. Also add a continue to the last if as well.

Reply:
@KamyarZiabari The design choice for using `elif` along with `continue` was discussed in PR [#256 ](https://github.com/CiscoTestAutomation/genieparser/pull/256#discussion_r493013132).